### ISSUE TITLE
500 error when attempting to lock user accounts

### ISSUE THREAD

**Issue by DanielTosello** [Karaage-Cluster/karaage#412]
django gives the following in logs: 

```
ERROR 2018-03-12 10:03:38,843 exception 152 140265335531264 Internal Server Error: /karaage/persons/detail/watchapon/lock/
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.6/site-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "/usr/local/lib/python3.6/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python3.6/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.6/contextlib.py", line 52, in inner
    return func(*args, **kwds)
  File "/usr/local/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/opt/karaage/karaage/people/views/persons.py", line 258, in lock_person
    person.lock()
  File "/opt/karaage/karaage/people/models.py", line 353, in lock
    self.save()
  File "/usr/local/lib/python3.6/site-packages/model_utils/tracker.py", line 171, in save
    ret = original_save(**kwargs)
  File "/opt/karaage/karaage/people/models.py", line 144, in save
    ua.lock()
  File "/opt/karaage/karaage/machines/models.py", line 293, in lock
    self.save()
  File "/usr/local/lib/python3.6/site-packages/model_utils/tracker.py", line 171, in save
    ret = original_save(**kwargs)
  File "/opt/karaage/karaage/machines/models.py", line 192, in save
    save_account(self)
  File "/opt/karaage/karaage/datastores/__init__.py", line 103, in save_account
    datastore.save_account(account)
  File "/opt/karaage/karaage/common/trace.py", line 541, in wrapper
    result = _func(*__argv, **__kwds)
  File "/opt/karaage/karaage/datastores/ldap.py", line 127, in save_account
    luser.save()
  File "/usr/local/lib/python3.6/site-packages/tldap/base.py", line 315, in save
    self._modify()
  File "/usr/local/lib/python3.6/site-packages/tldap/methods/__init__.py", line 139, in _modify
    super(baseMixin, self)._modify()
  File "/usr/local/lib/python3.6/site-packages/tldap/base.py", line 466, in _modify
    c.modify(self._dn, modlist, onfailure)
  File "/usr/local/lib/python3.6/site-packages/tldap/backend/fake_transactions.py", line 297, in modify
    return self._process(oncommit, onrollback, onfailure)
  File "/usr/local/lib/python3.6/site-packages/tldap/backend/fake_transactions.py", line 186, in _process
    result = self._do_with_retry(oncommit)
  File "/usr/local/lib/python3.6/site-packages/tldap/backend/base.py", line 144, in _do_with_retry
    return fn(self._obj)
  File "/usr/local/lib/python3.6/site-packages/tldap/backend/fake_transactions.py", line 292, in oncommit
    obj.modify(dn, modlist)
  File "/usr/local/lib/python3.6/site-packages/ldap3/core/connection.py", line 1051, in modify
    request = modify_operation(dn, changes, self.auto_encode, self.server.schema if self.server else None, validator=self.server.custom_validator if self.server else None, check_names=self.check_names)
  File "/usr/local/lib/python3.6/site-packages/ldap3/operation/modify.py", line 70, in modify_operation
    partial_attribute['vals'].setComponentByPosition(index, prepare_for_sending(validate_attribute_value(schema, attribute, value, auto_encode, validator, check_names=check_names)))
  File "/usr/local/lib/python3.6/site-packages/ldap3/protocol/convert.py", line 160, in validate_attribute_value
    raise LDAPInvalidValueError('value \'%s\' non valid for attribute \'%s\'' % (value, name))
ldap3.core.exceptions.LDAPInvalidValueError: value 'b'000001010000Z'' non valid for attribute 'pwdAccountLockedTime'
```

I assume this is a model datatype issue, and probably triggered by changes to underlying django functionality.

**Comment by brianmay** [comment]
I changed the formatting on your stack track so I can read it (need to use 3 backticks before and after).

**Comment by DanielTosello** [comment]
ugh sorry. Put it down to me rushing, it's not like the markup is different to most other places.

**Comment by brianmay** [comment]
https://github.com/cannatag/ldap3/issues/510

**Comment by brianmay** [comment]
This should be fixed by #420.

**Comment by brianmay** [comment]
Should be fixed now #422 has been merged.
